styleguide/js/zoolander.js

```diff
@@ -139,6 +141,70 @@ jQuery(document).ready(function($) {
   };
 }(jQuery)); //to protect and scope the JQuery alias = $
 
+//Rackspace Zoolander Tabs jQuery Plugin
+(function($){
+  'use strict';
+  $.fn.rsTabs = function(){
+    var $tabs = $(this);
+    var $zoolanderWindow = $(window);
+
+    $tabs.each(function(){
+      var $thisTabSet = $(this);
+      var $tabLink = $thisTabSet.find('.rsTabs-menu .rsTabs-tabItem');
+
+      //function to reset the tabs on page resizes
+      var resizeTimer;
+      $zoolanderWindow.resize(function(){
+        clearTimeout(resizeTimer);
+        resizeTimer = setTimeout(resizeResetTabs, 250);
+      });
+
+      //call back for the tabs reset
+      function resizeResetTabs() {
+        $tabLink.each(function(){
+          var $tL = $(this);
+          if($tL.hasClass('rsTabs-tabActive')){
+            $tL.siblings().removeClass('rsTabs-tabMobile');
+          }
+        });
+      }
+
+      //set first tab active
+      $thisTabSet.find('.rsTabs-menu .rsTabs-tabItem').first().addClass('rsTabs-tabActive');
+      $thisTabSet.find('.rsTabs-contentContainer div:first-child').addClass('rsTabs-tabActive');
+
+      var tabLinksAreVis = false;
+      $tabLink.click(function(){
+        var $li = $(this);
+        var $contentID = $li.find('a').prop('hash').split('#')[1];
+
+        //Set the clicked tab link active and remove active from siblings
+        $li.addClass('rsTabs-tabActive');
+        $li.siblings().removeClass('rsTabs-tabActive');
+
+        //Find mathcing content and show it / hide siblings
+        var $tabContent = $li.parent().parent().find('.rsTabs-contentContainer .' + $contentID);
+        $tabContent.addClass('rsTabs-tabActive').siblings().removeClass('rsTabs-tabActive');
+
+        //When clicked, check if mobile viewport, if so toggle other tab options
+        if (~~($zoolanderWindow.outerWidth()) < 769) {
+          if(tabLinksAreVis){
+            $li.siblings().removeClass('rsTabs-tabMobile');
+            tabLinksAreVis = false;
+          }else {
+            $li.siblings().addClass('rsTabs-tabMobile');
+            tabLinksAreVis = true;
+          }
+        }
+      });
+      $tabLink.find('a').click(function(e){
+        e.preventDefault();
+      });
+    });
+    return this;
+  };
+}(jQuery));
```

Comment:
I'm know we don't currently have it built into zoolander, but I'd really like to see some sort of testing around this functionality. Maybe we can add some end to end testing on this separately.